webapp/src/index.js

```diff
@@ -4,18 +4,18 @@ import AttachCommentToIssuePostMenuAction from 'components/post_menu_actions/att
 import AttachCommentToIssueModal from 'components/modals/attach_comment_to_issue';
 
 import CreateOrUpdateIssueModal from './components/modals/create_update_issue';
-import CloseOrReopenIssueModal from './components/modals/close_reopen_issue/close_reopen_issue.tsx';
+import CloseOrReopenIssueModal from './components/modals/close_reopen_issue/index.tsx';
```

Comment:
```suggestion
import CloseOrReopenIssueModal from './components/modals/close_reopen_issue;
```
When we don't specify a filename in the path, it takes the `index` file by default. Fix this wherever you've done this.